addons/player-thumb/userscript.js

```diff
@@ -0,0 +1,16 @@
+export default async function ({ addon, global, console }) {
+  const el = await addon.tab.waitForElement('div[class*="stage_green-flag-overlay-wrapper_"]', {
+    markAsSeen: true,
+  });
+
+  const projectId = window.location.pathname.split("/")[2];
+  const thumb = `https://uploads.scratch.mit.edu/get_image/project/${projectId}_480x360.png`;
```

Comment:
I think compatibility with the "high resolution thumbnails" addon should be considered.

Reply:
I don't think thumbnails are stored at higher resolutions that, but I could be wrong.

Reply:
Does http://uploads.scratch.mit.edu/get_image/project/1_960x720.png (for example) work if the image is a higher resolution then 480x360?

Reply:
Yes it does.

Reply:
I'm always using the highest resolution regardless if high resolution thumbnails is enabled or not now.